src/Command/ExportAssetsCommand.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use App\FileLogger;
+use App\SQL\FindAssets;
+use App\SQL\FindVariations;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * {description}
+ *
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ExportAssetsCommand extends Command
+{
+    private const ASSETS_CSV_FILENAME = 'assets.csv';
```

Comment:
this could be configured in a second time

Reply:
I agree